src/System.Memory/src/System/Buffers/Text/Base64Encoder.cs

```diff
@@ -135,7 +135,7 @@ public static OperationStatus EncodeToUtf8InPlace(Span<byte> buffer, int dataLen
             if (buffer.Length < encodedLength)
                 goto FalseExit;
 
-            int leftover = dataLength - dataLength / 3 * 3; // how many bytes after packs of 3
+            int leftover = dataLength - (dataLength / 3) * 3; // how many bytes after packs of 3
```

Comment:
Just to ensure that no compiler will "optimize" the `/ 3 * 3`.

Reply:
Parenthesis only ever clarify intent, [and are essentially removed at the parse stage](https://softwareengineering.stackexchange.com/questions/238786/does-c-compiler-remove-optimize-useless-parentheses).  "Optimizing" such a statement that way would completely break intent and integer math (consider what that process would do to `dataLength / 4 * 2`).

Now, for clarifying intent, it might be a benefit to have them there.